packages/javascript/bh-shared-ui/src/components/ManagedDatePicker/ManagedDatePicker.tsx

```diff
@@ -16,65 +16,30 @@
 
 import { DatePicker } from '@bloodhoundenterprise/doodleui';
 import { DateTime } from 'luxon';
-import { FC, useEffect, useId, useState } from 'react';
+import { FC, useEffect, useId, useRef, useState } from 'react';
 import { LuxonFormat, fiveYearsAgo, now } from '../../utils';
 
 const DATE_FORMAT_MASK = 'yyyy-mm-dd';
+const INVALID_DATE_MESSAGE = 'Input is not a valid date.';
 
-export const VALIDATIONS = {
```

Comment:
I kept running into edge cases so I took a different approach. The ManagedDatePicker now works with string values instead of Dates. It will only validate that the input is a valid date when the user types in the input. The validation to check end and start bounds is now handled in the DateRangeInput.

This is pretty much a full rewrite. It might be easier to just view the whole file rather than the changes.